docs/release-notes/cloud/cloud-2024-01-DD-gh-repository-ssh-keys-deprecation.md

```diff
@@ -0,0 +1,82 @@
+
+![Codacy GitHub App updated permissions request](../images/2023-12-DD-gh-updated-permissions.png)<!--TODO PLUTO-802 Rename image-->
+
+If you have any questions or need help, please contact <mailto:support@codacy.com>.
+
+## Removal of repository permissions for Administration and SSH keys {: id="timeline"}
+
+To ensure the conditions to use installation access tokens on GitHub organizations are met before the removal of repository permissions for Administration and SSH keys, Codacy will execute a phased rollout according to the timeline below:
+
+<table>
+  <thead>
+    <th>Date</th>
+    <th>Event</th>
+    <th>What to expect</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>January DD, 2024</td><!--TODO PLUTO-802 Update release date-->
+      <td>Codacy started using installation access tokens to clone and integrate with your repositories, and deprecated the usage of repository SSH keys</td>
+      <td>
+        <p>From this day on, repository SSH keys are used exclusively as a fallback mechanism when the Contents permission is missing.</p>
+        <p>If not done yet, make sure an organization owner <a href="#to-do">approves Codacy GitHub App updated permissions</a> on your GitHub organization.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>February 12, 2024</td>
+      <td>Brownout of repository permissions for Administration</td>
+      <td>
+        <p>On this day, Codacy won't use any fallback mechanism if the Contents permission is missing.</p>
+        <p>This will help you confirm that the updated permissions for the Codacy GitHub App were already approved for your organization.<br/>If on this date your Codacy quality analysis fails or you have problems adding new repositories to Codacy, make sure an organization owner <a href="#to-do">approves the updated permissions for the Codacy GitHub App</a> on your GitHub organization.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>February 19, 2024</td>
+      <td>Codacy will remove repository permissions for Administration from the Codacy GitHub App and drop the usage of repository SSH keys</td>
+      <td>
+        <p>Codacy will start using installation access tokens to clone and integrate with your repositories, with no fallback mechanisms. Also, repository permissions for Administration will be removed from the Codacy GitHub App.</p>
+        <p>If on this date the Codacy GitHub App updated permissions haven't been approved on your GitHub organization yet, your Codacy quality analysis will fail and you'll not be able to add new repositories to Codacy until an organization owner <a href="#to-do">approves the updated permissions.</a></p>
+      </td>
+    </tr>
+    <tr>
+      <td>To define</td>
+      <td>Codacy will no longer store repository SSH keys</td>
+      <td>
+        <p>On this day, Codacy will delete all the repository SSH keys stored in our systems.</p>
```

Comment:
Maybe we can just mention that users can delete Deploy keys themselves in GH 
https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-deploy-keys

for context link to your repo (I hope you have access): 
https://github.com/troubleshoot-codacy/test-app-analysis/settings/keys

Reply:
Yes, that is important information to mention. 🙏 
I added it, with a link to GH documentation.